browser/react/components/Login/Login.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+export default (props) => (
+  <div style={props.styles.container}>
+    <div>
+      <Link to="signup" style={{ 'textDecoration': 'none' }}>
+        <button key="signup" style={props.styles.signupButton}>Sign Up</button>
+      </Link>
+    </div>
+    <div style={props.styles.orDividerLineDiv}>
+      <div style={props.styles.orDividerLineBefore}></div>
+      <p style={props.styles.orDivider}></p>
+      <div style={props.styles.orDividerLineAfter}></div>
+    </div>
+    <form onSubmit={props.login}>
+      <div className="form-group">
```

Comment:
1. Is this `className`being used for something? 2. If so, can we move it to the inline styling?